pkgs/development/python-modules/mercantile/default.nix

```
@@ -0,0 +1,24 @@
+{ buildPythonPackage, lib, fetchFromGitHub, click, pytestCheckHook, hypothesis }:
```

dont feel strongly about it:
```suggestion
{ lib, buildPythonPackage, fetchFromGitHub
, click
, hypothesis
, pytestCheckHook
}:
```